README.md

```diff
@@ -12,38 +11,35 @@ Before starting the practical, make sure you have the following ready, if not, p
 * The URL [link](https://learning.ds-wizard.org/) for DSW training instance.
 * The [example scenario](https://github.com/elixir-luxembourg/DS-DM-training/blob/master/resources/DM-DP_RunningExample.pdf) for our workshop.
 
-## Exercice 1 : sign up
+## Exercice 1: sign up
 
 * Go to the url [link](https://learning.ds-wizard.org/) for DSW training and `Sign up`.
 
-
 ## Exercise 2: Create questionnaire (project) from existing knowledge model
 
 In the DSW tool, _Knowledge Models_ act as source databases that hold requirements in one or more sub-area of RDM.  
```

Comment:
This sentence is cool :D It is ok but I would be really lost reading and seeing this for the first time. Just to consider - no update needed